mandatory/4-tax.js

```diff
@@ -17,7 +19,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(amount) {
+  tax = (amount * 0.2) + (amount);
```

Comment:
```suggestion
  tax = calculateSalesTax(amount);
```

Reply:
This line will work as needed, but it is exactly the same as what you already implemented in calculateSalesTax function. Functions allow us to write code once and execute it multiple times.